Automattic/wp-calypso
# Add site overview started migration diy

Related to #95384
Based on https://github.com/Automattic/wp-calypso/pull/95497

## Proposed Changes

* It replaces the site overview screen when the migration started and it's the DIY option.

## Why are these changes being made?


* Inform the user that their migration started and what they can expect.

## Testing Instructions



* Make sure you have a site with the migration started status for the DIY - you can do it manually starting a migration through `/setup/migration` or you can add the sticker `migration-started-diy` manually (p1729023356708799/1728480467.656839-slack-C07N53UNUQJ).
* Navigate to the `/sites/` dashboard.
* Click on the site.
* Check that you see an overview specific for the migration, and this screen doesn't have a button to return to the flow.

## Screenshots

<img width="1146" alt="Screenshot 2024-10-24 at 19 24 12" src="https://github.com/user-attachments/assets/ab46fd4a-785e-4015-b1a8-a336d05b9241">

## Pre-merge Checklist
